### ISSUE TITLE
PRScs-auto

### ISSUE THREAD

**Issue by rameez500** [getian107/PRScs#53]
Hi Tian,

Hope you are doing well.

We have about 25,000 GWAS sample size and 2000 sample in target dataset. If I don’t want to create the validation and test dataset, is it reasonable to use PRS-CS-auto method?

The reason I am asking about this is because you mentioned in the github repository that we may need 100,000 GWAS sample size in PRSc-auto method: 

[https://github.com/getian107/PRScs] 
“PARAM_PHI (optional): Global shrinkage parameter phi. If phi is not specified, it will be learnt from the data using a fully Bayesian approach. This usually works well for polygenic traits with large GWAS sample sizes (hundreds of thousands of subjects).”

Thanks,
Rameez

**Comment by getian107** [comment]
Hi Rameez- Yes the auto version may not as accurate as fine-tuning the hyper-parameter but it should give you reasonable predictions.

**Comment by rameez500** [comment]
Hi Tian,

Thanks for the quick response!

If I like to use PRS-CS-grid method with multiple values for phi parameter, is it possible for you to tell me what sample size of target dataset is good for validation set in order to choose best parameter and remaining samples in the test set?

Thanks,
Rameez

**Comment by getian107** [comment]
Hi Rameez- The general principle is to have a sample size that can produce stable parameter and accuracy estimates in the validation and testing datasets. In practice whether a sample size is large enough or not depends on several factors and may vary from trait to trait and depend on the evaluation metric (e.g., estimating the OR at the tail of PRS compared to population average would often need a larger sample than calculating assessing the overall prediction accuracy using e.g. the variance explained or AUC). For 2000 samples in your case I think a half-half split sounds reasonable.
